discord_roles/discord_review_bot.py

```
@@ -450,6 +713,100 @@ async def get_course_list(self):
+            await interaction.response.send_message(
+                "🤖 Use `/link` inside a story-issues thread.",
+                ephemeral=True,
+            )
+            return
+
+        await interaction.response.defer(thinking=True)
+        try:
+            match = await self.lookup_story_issue_thread(
+                interaction.channel, hint=hint or ""
+            )
+        except Exception as err:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)